spring-boot-parent/pom.xml

```diff
@@ -473,7 +473,7 @@
 						<java.security.egd>file:/dev/./urandom</java.security.egd>
 						<java.awt.headless>true</java.awt.headless>
 					</systemPropertyVariables>
-					<argLine>-Xmx1024m -XX:MaxPermSize=256m</argLine>
+					<argLine>-Xmx1024m</argLine>
```

Comment:
Yes, it might break users using the parent on Java7 runtime. But I'm think in this pom.xml we use maven-enforcer-plugin to limit the java version is java8 or higher.

That means user can't use pom.xml on Java7 runtime no matter we have this option or not.